client/main.js

```diff
@@ -137,8 +137,8 @@ const routeOpts = {
       component: WorkflowTabs,
       props: ({ params }) => ({
         domain: params.domain,
-        runId: params.runId,
-        workflowId: params.workflowId,
+        runId: getEscapedForwardSlash(params.runId),
```

Comment:
runID is generated by cadence and is always uuid

Reply:
I will remove it if thats the case. Thanks!